tests/e2e/genesis.go

```diff
@@ -0,0 +1,110 @@
+package e2e
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+
+	"github.com/cosmos/cosmos-sdk/server"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	authtypes "github.com/cosmos/cosmos-sdk/x/auth/types"
+	banktypes "github.com/cosmos/cosmos-sdk/x/bank/types"
+	"github.com/cosmos/cosmos-sdk/x/genutil"
+	genutiltypes "github.com/cosmos/cosmos-sdk/x/genutil/types"
+	tmtypes "github.com/tendermint/tendermint/types"
+)
+
+func getGenDoc(path string) (*tmtypes.GenesisDoc, error) {
+	serverCtx := server.NewDefaultContext()
+	config := serverCtx.Config
+	config.SetRoot(path)
+
+	genFile := config.GenesisFile()
+	doc := &tmtypes.GenesisDoc{}
+
+	if _, err := os.Stat(genFile); err != nil {
```

Comment:
should we modify this to use `ioutils.ReadAll` just so as to reduce a few lines of code?